README.md

```diff
@@ -1,41 +1,85 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# FLAD-Mentorship app
 
-## Available Scripts
+FLAD Mentorship helps to link people to participate as a mentor or a mentee in the program [Let's talk about business](https://www.flad.pt/en/lets-talk-about-business/) from the Luso-American Development Foundation.
 
-In the project directory, you can run:
+## Development Notes
 
-### `npm start`
+FLAD-Mentorship app was bootstrapped with [Create React App](https://github.com/facebook/create-react-app). The project uses the Firebase Realtime Database and Storage as a backend, so it requires no server-side code or SQL queries.
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## To build the application
 
-The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
+The first step is to clone this repository in your local machine and then, open a terminal and cd into the project directory
 
-### `npm test`
+### 1. run `npm install`
 
-Launches the test runner in the interactive watch mode.<br>
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+This command will install all the necessary dependencies. [Material-ui](https://github.com/mui-org/material-ui), [firebase](https://github.com/firebase/), [react-to-print](https://www.npmjs.com/package/react-to-print) and [ramda](https://github.com/ramda/ramda) are some of them.
 
-### `npm run build`
+The file package.json contains the complete list of dependencies for this project.
 
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### 2. Create a project in Firebase
+
+In order to try this app you may create a new project in the Firebase Console. You can create a free account (here) [https://console.firebase.google.com/]
+
```

Comment:
The parenthesis and the square brackets are mixed up it has to be like this: `You can create a free account [here] (https://console.firebase.google.com/)` 😄

Reply:
Oh 😄! this one was trying to escape! Fixed it! 👍